src/commands/bitcount.json

```diff
@@ -54,24 +54,31 @@
                         "type": "integer"
                     },
                     {
-                        "name": "end",
-                        "type": "integer"
-                    },
-                    {
-                        "name": "unit",
-                        "type": "oneof",
+                        "name": "end-unit-block",
```

Comment:
Top of the file also has history, can you add a place-holder there for changing the arguments here. We don't know our next major version, but I would just say put 8.0 for now.

Reply:
I didn't add it intentially, because there is the version history (and since) at the top of the file and this change is not adding some args.

But I agree to your disagree, and now I think this (required -> optional) also being managed by history seems nice.
I'll update the History and commit it again

Reply:
Yeah. I had a very vague definition in the past of "When any change to arguments changes that may impact client implementations", which seems applicable here. We should add better documentation here around these json files.